apps/vr-tests-web-components/jest.config.cjs

```diff
@@ -0,0 +1,18 @@
+// @ts-check
+
+/**
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
+module.exports = {
```

Comment:
there  are no jest tests here, also using our root jest.preset which is for v9 packages might not be something we wanna do

Reply:
@miroslavstastny to add context here.

Reply:
Added by @TristanWatanabe in #28222. Perhaps copy&paste from existing FUIR9 screenshot tests.